integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/pages/openConditions/PreviousSentencesPage.groovy

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.cattool.pages.openConditions
+
+import geb.Page
+
+class PreviousSentencesPage extends Page  {
```

Comment:
Just thinking about how we will do this going forward... Would it be the QA who creates these groovy pages? 
Can the existing tests still pass without adding this new page?

Reply:
The tests failed without the page which is why I added it. I would prefer not to have to touch these tests at all.

Reply:
We need to discuss with Raza the way forward.